### PR TITLE
Add a ditado

### PR DESCRIPTION
**Pull request by pedromcunha**

### CODE REVIEW

File: ditados.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
       {
          "content":"É desde pequenino que se torce o pepino",
          "meaning":"Os bons costumes e educação que se dá as crianças deve ser feito desde o mais cedo possível."
+      },
+      {
+         "content":"Quem não arrisca, não petisca.",
+         "meaning":"As vezes correr um risco pode ter benefícios."
```
Comment (antonioalmeida) [inline ditados.json:9]: Missing an accent here on the capital 'A', i.e, À

**Review by antonioalmeida — CHANGES_REQUESTED** [review]

File: ditados.json (hunk 1 above)
Comment (pedromcunha) [reply ditados.json]: Ah, boa. Fixed :)

Comment (antonioalmeida) [reply ditados.json]: Sorry, wrong accent 😅

Comment (pedromcunha) [reply ditados.json]: Hah, sorry. Fixed it, this is what happens when I'm careless.

**Pull request merged**